lib/active_model_serializers/register_jsonapi_renderer.rb

```diff
@@ -22,43 +22,58 @@
 # render jsonapi: model
 #
 # No wrapper format needed as it does not apply (i.e. no `wrap_parameters format: [jsonapi]`)
-
 module ActiveModelSerializers::Jsonapi
   MEDIA_TYPE = 'application/vnd.api+json'.freeze
   HEADERS = {
     response: { 'CONTENT_TYPE'.freeze => MEDIA_TYPE },
     request:  { 'ACCEPT'.freeze => MEDIA_TYPE }
   }.freeze
+
+  def self.install
+    # actionpack/lib/action_dispatch/http/mime_types.rb
+    Mime::Type.register MEDIA_TYPE, :jsonapi
+    mime_type = Mime::Type.lookup(MEDIA_TYPE)
```

Comment:
At the time it seemed like this didn't work the same between Rails 4.x and 5. I went tried again this morning it works just fine.